cross/go2rtc/Makefile

```diff
@@ -0,0 +1,31 @@
+PKG_NAME = go2rtc
+PKG_VERS = 1.6.2
+PKG_EXT = tar.gz
+PKG_DIST_SITE = https://github.com/AlexxIT/go2rtc/archive/refs/tags/v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_FILE = v$(PKG_VERS).$(PKG_EXT)
+EXTRACT_PATH = $(WORK_DIR)/src/github.com/$(PKG_NAME)
```

Comment:
You should either extract to the final destination (without needing a post_extract target)
- OR - 
Implement post_extract target in a way that does not need a special EXTRACT_PATH

Reply:
this code copied from `minio` package...

Reply:
ok, leave it as is...